targets/default.nix

```diff
@@ -37,6 +38,52 @@
     import ./nvidia-jetson-orin.nix {inherit jetpack-nixos microvm;}
     // {
       format = "raw-efi";
+      # XXX: This override is needed to reduce overall image size, but will be applied only for package and not for nixos configuration.
+      pkgs = let
+        overlay = self: super: {
+          qemu = super.qemu.override {
+            alsaSupport = false;
+            pulseSupport = false;
+            sdlSupport = false;
+            jackSupport = false;
+            gtkSupport = false;
+            vncSupport = false;
+            smartcardSupport = false;
+            spiceSupport = false;
+            ncursesSupport = false;
+            libiscsiSupport = false;
+            tpmSupport = false; # XXX: Probably shouldn't be disabled
+            hostCpuOnly = true;
+          };
+          git = super.git.override {
```

Comment:
I'd like to see git on image via `modules/development/essentials.nix` but this is good for now.

We can mark it as `# XXX` (or `# TODO:`) for now. 

On most minimal host profile git should not be needed - even for updates.

Reply:
I’ve done overlay for git because of microvm. It uses it internally in the host nixos module, and with this overlay, image size was reduced by 200mb. So it is actually unrelated to the development environment, but if you want to add git to devenv you should be aware of this change.

It also can be done with a little hack for microvm module, by overlaying packages especially for it. But for now, let it be this way.